### PR TITLE
Added pre-3.10 kraken source code, README, and config files

### PR DESCRIPTION
**Pull request by wabouhamad**
Kraken is a resiliency tool for OpenShift which can crash a node,
master, etcd and checks if the cluster recovers from the failures.
This is the tool version that is supported for pre-3.10 OCP version.
- created kraken directory with subdirectories, source files, config files and png image files

### CODE REVIEW

**Comment by wabouhamad** [comment]
@chaitanyaenr PTAL, this is the original code with updated Openshift-Labeler label.yaml playbook.

File: openshift_tooling/kraken/setup-env.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+fi
+if [[ $(echo $GOPATH) == "" ]];then
+	mkdir -p $gopath_dir/.go
+	echo "GOPATH=$gopath_dir/.go" >> ~/.bashrc
+	echo "export GOPATH" >> ~/.bashrc
+	echo "PATH=\$PATH:\$GOPATH/bin # Add GOPATH/bin to PATH for scripting" >> ~/.bashrc
+	source ~/.bashrc
+fi
+# get kube-monkey
+go get github.com/asobti/kube-monkey
```
Comment (chaitanyaenr) [inline openshift_tooling/kraken/setup-env.sh]: @wabouhamad we don't need this script as we are not using kube-monkey in our tool.

Comment (wabouhamad) [reply openshift_tooling/kraken/setup-env.sh]: @chaitanyaenr, removed setup-env.sh.  Should be ready for merge.

**Comment by chaitanyaenr** [comment]
Thanks for the PR @wabouhamad.

**Pull request merged**